### PR TITLE
General openroad tool driver cleanup

### PR DESCRIPTION
**Pull request by gadfort**
This fixes a few minor issues
Reorganizes the parameter setting to allow for getting them from the PDK and library settings
Adds some missing flags for resizer, (depends on https://github.com/siliconcompiler/zerosoc/pull/27)

### CODE REVIEW

File: siliconcompiler/tools/openroad/openroad.py [hunk 1]
```diff
@@ -653,6 +481,407 @@ def build_pex_corners(chip):
...
+    _set_parameter(chip, param_key='cts_clock_buffer',
+                   default_value=chip.get('library', mainlib, 'asic', 'cells', 'clkbuf',
+                                          step=step, index=index)[-1],
+                   schelp='buffer to use during clock tree synthesis')
+    _set_parameter(chip, param_key='cts_distance_between_buffers',
+                   default_value='100',
+                   schelp='maximum distance between buffers during clock tree synthesis in microns')
+    _set_parameter(chip, param_key='cts_cluster_diameter',
+                   default_value='100',
+                   schelp='clusting distance to use during clock tree synthesis in microns')
```
Comment (nmoroze) [inline siliconcompiler/tools/openroad/openroad.py:708]: typo: "clusting"

File: siliconcompiler/tools/openroad/openroad.py [hunk 2]
```diff
@@ -653,6 +481,407 @@ def build_pex_corners(chip):
...
+
+    pdk_min_layer = chip.get('pdk', pdkname, 'minlayer', stackup)
+    pdk_max_layer = chip.get('pdk', pdkname, 'maxlayer', stackup)
+
+    _set_parameter(chip, param_key='grt_use_pin_access',
+                   default_value='false',
+                   schelp='true/false, when true perform pin access before global routing')
+    _set_parameter(chip, param_key='grt_overflow_iter',
+                   default_value='100',
+                   schelp='maximum number of iterations to use in flobal routing when '
```
Comment (nmoroze) [inline siliconcompiler/tools/openroad/openroad.py:729]: typo: "flobal"

**Review by nmoroze — APPROVED** [review]
LGTM, nice cleanup. Did find a few typos that were copy-pasted forward that we probably might as well fix now.

File: siliconcompiler/tools/openroad/openroad.py (hunk 2 above)
Comment (gadfort) [reply siliconcompiler/tools/openroad/openroad.py:729]: I'm going to coin that term :-)

**Pull request merged**